ramp4-pcar4/ramp4-pcar4
# Fix CSS on sample page

### Related Item(s)
Issue #2368 

### Changes
- [FIX] Ensures that the header for sample pages resizes properly on mobile sizes, preventing white areas on the right and bottom of the page.

### Testing
Steps:
1. Open any sample, then use the DevTools to open the device emulator.
2. Resize the page to a mobile size, and play with the sizes after that. Notice that the header will grow, shrink, add horizontal scrollbars, etc. to ensure everything remains usable, and ensuring the page always takes up the entire screen size.
